packages/ui/.storybook/config.js

```diff
@@ -47,7 +47,7 @@ addDecorator(
      * show addon panel as a vertical panel on the right
      * @type {Boolean}
      */
-    addonPanelInRight: true,
+    addonPanelInRight: false,
```

Comment:
This moves the tools panel to the bottom, which is a bit handier if your browser isn't running fullscreen. This can be reverted if we liked the tool on the right more.